spec/accounting/api/accounting_api_spec.rb

```diff
@@ -32,6 +32,53 @@
     end
   end
 
+  describe 'bank statement report' do
```

Comment:
Must feel good to write some rspec again 😬

Reply:
look at these tests. bloody art. :tada: